decidim-conferences/app/cells/decidim/conferences/conference_m/tags.erb

```diff
@@ -0,0 +1 @@
+ <%= render_hook :upcoming_meeting_for_card %>
```

Comment:
Shouldn't `tags.erb` contain a list of tags instead of a list of upcoming meetings?

Reply:
Probably. But then assembly should also be changed...
Maybe @mrcasals or @agustibr can give more info about that

Reply:
In `Proposals` tags refer to `Category` and `Scope` :information_desk_person:

Reply:
@oriolgual Because the 'tags.erb' cell is also used to show the upcoming meetings, no?

Reply:
Oh, I didn't know, OK.

Reply:
So, @oriolgual @decidim/lot-core What nomenclature do we follow to solve this?

Reply:
You can:

- override the cell's `show.erb` file to change the name of the partial
- rename the `tags.erb` section from all the M Card cells to something else
- leave it as it is

I think leaving it as it is, even though it can lead to confusion, is good enough, but if you prefer any other option then go for it! 😄

Reply:
for now I leave it as it is. I propose to open a new issue / PR to change it wherever it is necessary